src/main/java/com/couchbase/lite/MutableDocument.java

```diff
@@ -90,7 +90,10 @@ protected MutableDocument(Document doc) {
         }
     }
 
-    MutableDocument(String id, Document doc) { this(doc.getDatabase(), id, doc.getC4doc()); }
+    MutableDocument(String id, Document doc) {
+        this(doc.getDatabase(), id, null);
+        setData(doc.getContent().toMap());
+    }
```

Comment:
I think it's better to implement it the same way as the `protected MutableDocument(Document doc)` method is doing so that it doesn't need to convert all data into Map.

Reply:
Yes.  This change was simple, quick, and it worked, so appropriate for a release branch patch.
I've filed https://issues.couchbase.com/browse/CBL-241 to fix it.